websites/R/RadiosNet/presence.ts

```diff
@@ -0,0 +1,124 @@
+const presence = new Presence({
+    clientId: "1221866611611402363",
+});
+const browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum AssetsRadios {
+    Logo = "https://static.radios.com.br/img/logo-radiosnet-512-solid.png",
+}
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+        largeImageKey: AssetsRadios.Logo,
+        startTimestamp: browsingTimestamp,
+    };
+
+    if (document.location.pathname === "/")
+        presenceData.details = "Página Inicial";
+    else if (document.location.pathname.includes("/favoritos")) {
+        presenceData.details = "Favoritos";
+        presenceData.smallImageKey = Assets.Reading;
+    } else if (document.location.pathname.includes("/lista/pais/brasil/33")) {
+        presenceData.details = "Rádios do Brasil";
+        presenceData.smallImageKey = Assets.Search;
+    } else if (document.location.pathname.includes("/lista/pais")) {
+        const hasCountry =
+            document.querySelector(".page-header")?.innerHTML;
+        if (hasCountry === "Lista de países") {
+            presenceData.details = "Rádios Internacionais";
+            presenceData.smallImageKey = Assets.Search;
+        } else {
+            const countryName = document
+                .querySelector(".page-header")?.innerHTML.split("<small>")[1].split("</small>")[0];
```

Comment:
Check if these can be simplified to a different query selector, or done using `textContent`